src/rod/__init__.py

```diff
@@ -51,12 +51,42 @@ def _is_editable():
     return rod_package_dir not in site.getsitepackages()
 
 
-# Initialize the logging verbosity depending on the installation mode.
-logging.configure(
-    level=logging.LoggingLevel.DEBUG if _is_editable() else logging.LoggingLevel.WARNING
-)
+def get_default_logging_level(env_var: str) -> logging.LoggingLevel:
+    """
+    Get the default logging level.
+
+    Args:
+        env_var: The environment variable to check.
+
+    Returns:
+        The logging level to set.
+    """
+
+    import os
+
+    try:
+        return logging.LoggingLevel[os.environ[env_var].upper()]
+
+    # Raise if the environment variable is set but the logging level is invalid.
+    except AttributeError:
+        msg = f"Invalid logging level defined in {env_var}: '{os.environ[env_var]}'"
+        raise ValueError(msg)
```

Comment:
```suggestion
    except AttributeError as exc:
        msg = f"Invalid logging level defined in {env_var}: '{os.environ[env_var]}'"
        raise ValueError(msg) from exc
```
Why changing the exception type when raising?

```suggestion
    except AttributeError as exc:
        msg = f"Invalid logging level defined in {env_var}: '{os.environ[env_var]}'"
        raise exc(msg)
```

Reply:
I didn't even noticed that I changed the type :) good catch

Reply:
I actually never used chained exceptions. After reading the rationale behind it, it seems to me that in this case it makes sense that the original exception is `AttributeError`, so that is clear that the root problem is the access of a non-existent enum field, and the final exception is a `Runtime` error that provides more details.

Reply:
Yes, I agree with you that the second option seems more correct and straightforward

Reply:
I simplified the exceptions further in https://github.com/ami-iit/rod/pull/35/commits/44068bcc263b8c50c343ca0dc6a9d75d3d04215d.